File: server/plugin/mock_subscribe/router/callback.go
package router

import "github.com/gin-gonic/gin"

type callback struct{}

func (r *callback) Init(public *gin.RouterGroup, private *gin.RouterGroup) {
	_ = private

	public.POST("papay/notify", apiInfo.Callback.ReceiveContract)
	public.GET("papay/callback-records", apiInfo.Callback.GetCallbackRecordList)
	public.GET("papay/callback-record", apiInfo.Callback.FindCallbackRecord)
}

File: web/src/plugin/mock_subscribe/api/callback.js
import service from '@/utils/request'

export const getCallbackRecordList = (params) => {
  return service({
    url: '/papay/callback-records',
    method: 'get',
    params
  })
}

export const findCallbackRecord = (params) => {
  return service({
    url: '/papay/callback-record',
    method: 'get',
    params
  })
}
